feature/groups/src/main/java/com/mifos/feature/groupsList/presentation/GroupsListState.kt

```diff
@@ -0,0 +1,16 @@
+package com.mifos.feature.groupsList.presentation
+
+import androidx.paging.PagingData
+import com.mifos.core.objects.group.Group
+import kotlinx.coroutines.flow.Flow
+
+sealed interface GroupsListState {
+
+    data object Empty: GroupsListState
+
+    data class Error(val message: String): GroupsListState
+
+    data class GroupsFromAPI(val groups: Flow<PagingData<Group>>): GroupsListState
+
+    data class GroupsFromLocalDB(val groups: List<Group>): GroupsListState
```

Comment:
groups should be same type, this is data source UI should not know from where data is comming

Reply:
Yes, it should be same type and the UI Layer should be independent, will fix in next commit